integration-tests/src/test/groovy/com/okta/sdk/tests/it/FactorsIT.groovy

```diff
@@ -178,6 +188,60 @@ class FactorsIT extends ITSupport {
         assertThat response.getFactorResult(), is(VerifyUserFactorResponse.FactorResultEnum.SUCCESS)
     }
 
+    @Test
+    void testEmailUserFactor() {
+        User user = randomUser()
+        assertThat user.listFactors(), emptyIterable()
+
+        EmailUserFactor emailUserFactor = client.instantiate(EmailUserFactor)
+            .setFactorType(FactorType.EMAIL)
+            .setProvider(FactorProvider.OKTA)
+            .setProfile(client.instantiate(EmailUserFactorProfile)
+                .setEmail(user.getProfile().getEmail()))
+
+        assertThat emailUserFactor.id, nullValue()
+        // enroll and activate
+        assertThat emailUserFactor, sameInstance(user.enrollFactor(emailUserFactor, false, null, null, true))
+        assertThat emailUserFactor.getStatus(), is(FactorStatus.ACTIVE)
+        assertThat emailUserFactor.id, notNullValue()
+
+        VerifyFactorRequest request = client.instantiate(VerifyFactorRequest)
+        VerifyUserFactorResponse response = emailUserFactor.verify(request, null, null)
+        assertThat response.getFactorResult(), is(VerifyUserFactorResponse.FactorResultEnum.CHALLENGE)
+    }
+
+    @Test
+    void testU2fUserFactorCreation() {
+        User user = randomUser()
+        assertThat user.listFactors(), emptyIterable()
+
+        U2fUserFactor u2fUserFactor = client.instantiate(U2fUserFactor)
+            .setFactorType(FactorType.U2F)
+            .setProvider(FactorProvider.GOOGLE)
+            .setProfile(client.instantiate(U2fUserFactorProfile))
+
+        assertThat u2fUserFactor.id, nullValue()
+        assertThat u2fUserFactor, sameInstance(user.enrollFactor(u2fUserFactor)) //TODO: check why Okta core responds with -  HTTP 400, Okta E0000001 (Api validation failed: factorEnrollRequest - Invalid Factor.), ErrorId oaeV_agqOzrSL-dXGwvdDLpSg
```

Comment:
@vijetmahabaleshwar-okta Looks like the factor provider `Google` needs to be enabled in Oktapreview Org?